---
bool ok
string errstr
ControllerInfo active_controller
float64[] goal
float64[] actual
